### PR TITLE
Add bindings for more X509_VERIFY_PARAM functions

### PR DESCRIPTION
**Pull request by steffen-eiden**
Signed-off-by: Steffen Eiden <seiden@linux.ibm.com>

### CODE REVIEW

**Comment by steffen-eiden** [comment]
clippy is complaining about some unrelated issues.

File: openssl/src/x509/verify.rs [hunk 1]
```diff
@@ -162,4 +162,48 @@ impl X509VerifyParamRef {
     pub fn set_depth(&mut self, depth: c_int) {
         unsafe { ffi::X509_VERIFY_PARAM_set_depth(self.as_ptr(), depth) }
     }
+
+    /// Sets the authentication security level to auth_level
+    #[corresponds(X509_VERIFY_PARAM_set_auth_level)]
+    #[cfg(any(ossl110))]
```
Comment (sfackler) [inline openssl/src/x509/verify.rs:168]: `any(ossl110)` is equivalent to just `ossl110`.

**Comment by sfackler** [comment]
Clippy should be fixed on master if you merge/rebase.

File: openssl/src/x509/verify.rs [hunk 2]
```diff
@@ -162,4 +162,48 @@ impl X509VerifyParamRef {
...
+    #[corresponds(X509_VERIFY_PARAM_set_auth_level)]
+    #[cfg(ossl110)]
+    pub fn set_auth_level(&mut self, lvl: c_int) {
+        unsafe { ffi::X509_VERIFY_PARAM_set_auth_level(self.as_ptr(), lvl) }
+    }
+
+    /// Gets the current authentication security level
+    #[corresponds(X509_VERIFY_PARAM_get_auth_level)]
+    #[cfg(ossl110)]
+    pub fn get_auth_level(&self) -> i32 {
```
Comment (sfackler) [inline openssl/src/x509/verify.rs:176]: Getters are usually don't have the `get` prefix.

**Pull request merged**